app/medInria/medHomepageArea.cpp

```diff
@@ -284,33 +314,9 @@ medHomepageArea::medHomepageArea ( QWidget * parent ) : QWidget ( parent ), d (
     d->stackedWidget->addWidget(d->aboutWidget);
     d->stackedWidget->addWidget(d->pluginWidget);
     d->stackedWidget->addWidget(d->settingsWidget);
+    d->stackedWidget->addWidget(d->logWidget);
     d->stackedWidget->setCurrentIndex(0);//d->infoWidget
     d->stackedWidget->setSizePolicy(QSizePolicy::Maximum,QSizePolicy::Maximum);
-
-    //Setup homepage animations
```

Comment:
I liked that animation :(.
I don't understand the current trend where all aesthetics are minimised . Even having shadows on buttons now is considered obsolete (https://en.wikipedia.org/wiki/Flat_design). Everything has to be flat, 2D, unanimated, undecorated, with no life in it.
Groumph.